llvm/lib/Analysis/CmpInstAnalysis.cpp

```diff
@@ -165,3 +165,18 @@ llvm::decomposeBitTestICmp(Value *LHS, Value *RHS, CmpInst::Predicate Pred,
 
   return Result;
 }
+
+std::optional<DecomposedBitTest>
+llvm::decomposeBitTest(Value *Cond, bool LookThruTrunc, bool AllowNonZeroC) {
+  using namespace PatternMatch;
```

Comment:
Unused?

Reply:
yes the using is unused forgot to remove it when I have removed the trunc handling that I hade here. and will add back soon but have removed it now